Riot/Modules/Room/Location/LocationMarkerView.swift

```diff
@@ -20,14 +20,23 @@ import Mapbox
 
 class LocationMarkerView: MGLAnnotationView, NibLoadable {
     
+    @IBOutlet private var markerBackground: UIImageView!
     @IBOutlet private var avatarView: UserAvatarView!
     
+    private static var usernameColorGenerator = UserNameColorGenerator()
+    private let theme: Theme = ThemeService.shared().theme
+    
     override func awakeFromNib() {
         super.awakeFromNib()
         translatesAutoresizingMaskIntoConstraints = false
     }
     
     func setAvatarData(_ avatarData: AvatarViewDataProtocol) {
+        Self.usernameColorGenerator.defaultColor = theme.colors.primaryContent
```

Comment:
We can probably inject the color from the outside by adding either a `let avatarBackgroundColor: UIColor` property or injecting the UserNameColorGenerator in the init or as var `var usernameColorGenerator: UserNameColorGenerator`